src/Analyzers/VisualBasic/Analyzers/UseAutoProperty/VisualBasicUseAutoPropertyAnalyzer.vb

```diff
@@ -24,43 +34,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseAutoProperty
             Return True
         End Function
 
-        Protected Overrides Sub AnalyzeCompilationUnit(context As SemanticModelAnalysisContext, root As SyntaxNode, analysisResults As List(Of AnalysisResult))
-            AnalyzeMembers(context, DirectCast(root, CompilationUnitSyntax).Members, analysisResults)
-        End Sub
-
-        Private Sub AnalyzeMembers(context As SemanticModelAnalysisContext,
-                                   members As SyntaxList(Of StatementSyntax),
-                                   analysisResults As List(Of AnalysisResult))
-            For Each member In members
-                AnalyzeMember(context, member, analysisResults)
-            Next
-        End Sub
-
-        Private Sub AnalyzeMember(context As SemanticModelAnalysisContext,
-                                  member As StatementSyntax,
-                                  analysisResults As List(Of AnalysisResult))
-
-            If member.Kind() = SyntaxKind.NamespaceBlock Then
-                Dim namespaceBlock = DirectCast(member, NamespaceBlockSyntax)
-                AnalyzeMembers(context, namespaceBlock.Members, analysisResults)
-            End If
-
-            ' If we have a class or struct or module, recurse inwards.
-            If member.IsKind(SyntaxKind.ClassBlock) OrElse
-               member.IsKind(SyntaxKind.StructureBlock) OrElse
-               member.IsKind(SyntaxKind.ModuleBlock) Then
-
-                Dim typeBlock = DirectCast(member, TypeBlockSyntax)
-                AnalyzeMembers(context, typeBlock.Members, analysisResults)
-            End If
-
-            Dim propertyDeclaration = TryCast(member, PropertyBlockSyntax)
-            If propertyDeclaration IsNot Nothing Then
-                AnalyzeProperty(context, propertyDeclaration, analysisResults)
-            End If
-        End Sub
```

Comment:
no need to walk down.  we're already starting on a named tpe, and we're registering for its properties.